markets/bfp-market/contracts/modules/OrderModule.sol

```diff
@@ -193,34 +198,37 @@ contract OrderModule is IOrderModule {
         );
 
         PerpMarket.Data storage market = PerpMarket.exists(marketId);
+        Runtime_commitOrder memory runtime;
 
         if (market.orders[accountId].sizeDelta != 0) {
             revert ErrorUtil.OrderFound();
         }
 
         validateOrderHooks(hooks);
 
-        uint256 oraclePrice = market.getOraclePrice();
+        runtime.oraclePrice = market.getOraclePrice();
 
         PerpMarketConfiguration.Data storage marketConfig = PerpMarketConfiguration.load(marketId);
-
+        runtime.params = Position.TradeParams(
```

Comment:
Is it super egregious to store structs within structs wrt gas? Can this just not be part of the runtime?

Reply:
Yea I can try but I think I did get stack too deep without this. And there's no other non storage vars.. We do the same in `settleOrder`

Reply:
Yea not sure how else I can strcuture it